oxidation/client/src/components/SettingsModal.vue

```diff
@@ -0,0 +1,344 @@
+              <!-- synchro wifi -->
+              <settings-option
+                :title="$t('SettingsPage.synchroWifiOnly')"
+                :description="$t('SettingsPage.synchroWifiOnlyDescription')"
+                v-model="config.synchroWifiOnly"
+              />
+              <!-- change lang -->
+              <!-- dropdown needs to be change depending on this issue : https://github.com/Scille/parsec-cloud/issues/4527 -->
+              <ion-item>
+                <ion-select
+                  interface="popover"
+                  :selected-text="$t(`SettingsPage.lang.${$i18n.locale.replace('-', '')}`)"
+                  :label="$t('SettingsPage.language')"
+                  @ion-change="changeLang($event.detail.value)"
+                >
+                  <ion-select-option value="en-US">
+                    {{ $t('SettingsPage.lang.enUS') }}
+                  </ion-select-option>
+                  <ion-select-option value="fr-FR">
+                    {{ $t('SettingsPage.lang.frFR') }}
+                  </ion-select-option>
+                </ion-select>
+              </ion-item>
+              <!-- change theme -->
+              <!-- dropdown needs to be change depending on this issue : https://github.com/Scille/parsec-cloud/issues/4527 -->
+              <ion-item>
+                <ion-select
+                  interface="popover"
+                  :value="config.theme"
+                  :label="$t('SettingsPage.theme.label')"
+                  @ion-change="changeTheme($event.detail.value)"
+                >
+                  <ion-select-option value="dark">
+                    {{ $t('SettingsPage.theme.dark') }}
+                  </ion-select-option>
+                  <ion-select-option value="light">
+                    {{ $t('SettingsPage.theme.light') }}
+                  </ion-select-option>
+                  <ion-select-option value="system">
+                    {{ $t('SettingsPage.theme.system') }}
+                  </ion-select-option>
+                </ion-select>
+              </ion-item>
+            </ion-list>
+          </div>
+          <!-- advanced -->
+          <div
+            v-else
+            class="settings-advanced"
+          >
+            <ion-list class="settings-list">
+              <!-- send error report -->
+              <settings-option
+                :title="$t('SettingsPage.enableTelemetry')"
+                :description="$t('SettingsPage.enableTelemetryDescription')"
+                v-model="config.enableTelemetry"
+              />
+              <!-- minimise in status bar -->
+              <settings-option
+                :title="$t('SettingsPage.minimizeToSystemTray')"
+                :description="$t('SettingsPage.minimizeToSystemTrayDescription')"
+                v-model="config.minimizeToTray"
+              />
+              <!-- display unsync files -->
+              <settings-option
+                :title="$t('SettingsPage.unsyncFiles')"
```

Comment:
`SettingsPage.unsyncFiles` and `SettingsPage.unsyncFilesDescription` seem to be missing from both translation files.

Reply:
replaced by `meteredConnection` and the translation files are added